### PR TITLE
Shows All the channels for bundles and can unlink them easily

### PR DESCRIPTION
**Pull request by yuvrajjsingh0**
/claim #255

### CODE REVIEW

**Comment by WcaleNieWolny** [comment]
Looks good, there are no tests but i could add them in the morning

**Comment by riderx** [comment]
Can you share screenshots ?

**Comment by yuvrajjsingh0** [comment]
> Can you share screenshots ?

Screenshot 2023-12-03 032619
On Click on any channel: 
Screenshot 2023-12-03 032802
On unlink:
Screenshot 2023-12-03 032817

**Comment by WcaleNieWolny** [comment]
image

I am not a big fan of the spacing, it just looks out of place. Could you move it to the left so it matches the rest of the table?

Also I found a bug in your code. I enabled AB testing 
image

And set the same bundle for both version A and version B
image

Then I checked the bundle page and only 1 of those channels showed up
image

The desired behavior would be to have `two_default-B, two_default-A` (Order does not matter)

Given this setup I tried to link the same bundle to the `no_access` channel. It worked so I tried to unlink it but unfortunately I was unable to

https://github.com/Cap-go/capgo/assets/50914789/c4037967-d274-4ca7-a857-7d9e4696459a

After disabling AB testing everything worked correctly 

https://github.com/Cap-go/capgo/assets/50914789/73915ddf-4a6a-4da3-be6e-13518b62fd38

**Comment by yuvrajjsingh0** [comment]
I have fixed these bugs, here's the video of same 

https://github.com/Cap-go/capgo/assets/20877577/d84aa36a-4011-40c5-aa49-cdcb4d91e0d8

File: supabase/functions/_utils/lemon.ts
Comment (WcaleNieWolny) [inline supabase/functions/_utils/lemon.ts:1]: why is this file here?

Comment (yuvrajjsingh0) [reply supabase/functions/_utils/lemon.ts:1]: I have removed this file. I was working on some other issue and stashed it

**Comment by yuvrajjsingh0** [comment]
@riderx any update on this?
